rpc/debug_api.go

```diff
@@ -0,0 +1,109 @@
+package rpc
+
+import (
+	"context"
+	"io/ioutil"
+	"time"
+	"os"
+	"net/http"
+	"net/url"
+	"encoding/json"
+	"errors"
+
+	"github.com/sirupsen/logrus"
+	"github.com/patrickmn/go-cache"
+
+	web3Types "github.com/openweb3/web3go/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/scroll-tech/rpc-gateway/util/rpc/handlers"
+)
+
+var (
+	whiteListURL     string
+	whiteListCache = cache.New(10 * time.Minute, 10 * time.Minute)
```

Comment:
We can probably use a large timeout, maybe 2 hours, what do you think? @silathdiir

Reply:
It seems that we may not use a long expired time for this cache. Because we cached records of `LeanCloud` tables (Debugger, Deployer and Subscriber) in [testnet/whitelist-backend](https://github.com/scroll-tech/testnet/tree/staging/whitelist-backend), and it exports [some refresh APIs](https://github.com/scroll-tech/testnet/pull/249) to refresh whitelist-backend cache immediately, which are used when user adds a new `Deployer` to `LeanCloud` and suppose to make it available in a short time.

It seems that with a large timeout of this roc-gateway cache it makes hard to refresh it when we want to make a new `Deployer` available in a short time.